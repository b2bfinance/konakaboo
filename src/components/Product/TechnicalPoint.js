import React from 'react';
import styled from 'styled-components';
import { queries } from '../../utils/media';
import CheckCircleIcon from '@material-ui/icons/CheckCircle';

export const Wrapper = styled.li`
  align-items: center;
  display: flex;
  font-size: 1.2rem;
  margin-bottom: 1rem;

  ${queries.desktop`
    margin-bottom: .5rem;
    padding-left: 1rem;
  `};

  svg {
    fill: ${props => props.theme.infoCheckColor};
    margin-right: 0.5rem;
  }
`;

export default ({ points, theme }) =>
  points.map((point, i) => (
    <Wrapper key={i} theme={theme}>
      <CheckCircleIcon />
      {point}
    </Wrapper>
  ));
