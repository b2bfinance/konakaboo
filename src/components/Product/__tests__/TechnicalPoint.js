import React from 'react';
import { render } from 'enzyme';
import TechnicalPoint from '../TechnicalPoint';

describe('TechnicalPoint', () => {
  test('renders correctly', () => {
    expect(
      render(<TechnicalPoint points={['foo', 'bar']} theme={stubData.theme} />)
    ).toMatchSnapshot();
  });
});
