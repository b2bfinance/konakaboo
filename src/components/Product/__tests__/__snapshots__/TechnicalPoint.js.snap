// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TechnicalPoint renders correctly 1`] = `
Array [
  .c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 1.2rem;
  margin-bottom: 1rem;
}

.c0 svg {
  fill: #4dc16f;
  margin-right: 0.5rem;
}

@media (min-width:62em) {
  .c0 {
    margin-bottom: .5rem;
    padding-left: 1rem;
  }
}

<li
    class="c0"
>
    <svg
        aria-hidden="true"
        class="MuiSvgIcon-root-1"
        focusable="false"
        viewBox="0 0 24 24"
    >
        <path
            d="M0 0h24v24H0z"
            fill="none"
        >
            
        </path>
        <path
            d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-2 15l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
        >
            
        </path>
    </svg>
    foo
</li>,
  .c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 1.2rem;
  margin-bottom: 1rem;
}

.c0 svg {
  fill: #4dc16f;
  margin-right: 0.5rem;
}

@media (min-width:62em) {
  .c0 {
    margin-bottom: .5rem;
    padding-left: 1rem;
  }
}

<li
    class="c0"
>
    <svg
        aria-hidden="true"
        class="MuiSvgIcon-root-1"
        focusable="false"
        viewBox="0 0 24 24"
    >
        <path
            d="M0 0h24v24H0z"
            fill="none"
        >
            
        </path>
        <path
            d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-2 15l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
        >
            
        </path>
    </svg>
    bar
</li>,
]
`;
